src/components/Bookmark/Row.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import {
+  ActivityIndicator,
+  Alert,
+  AsyncStorage,
+  Dimensions,
+  Image,
+  StyleSheet,
+  TouchableHighlight,
+  TouchableWithoutFeedback,
+  View,
+} from 'react-native';
+import { Actions } from 'react-native-router-flux';
+import { dimensions } from '../Shared/Dimensions';
+import Text from '../Shared/UniText';
+
+class Row extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = {
+      profileImage: '',
+      name: '',
+      position: '',
+      company: '',
+      currentStatus: '',
+      loaded: false,
+    };
+  }
+  
+  componentDidMount() {
+    this.setState({
+      profileImage: this.getProfileImage(),
+      name: this.getName(),
+      currentStatus: this.getCurrentStatus(),
+      loaded: true,
+    });
+  }
+  
+  getProfileImage() {
+    let image;
+    if (this.props.dataSource.profile_picture) {
+      image = { uri: this.props.dataSource.profile_picture_small ?
+        this.props.dataSource.profile_picture_small : this.props.dataSource.profile_picture };
+      return image;
+    } else {
+      image = require('../../resources/pattern.png');
+      return image;
+    }
+  }
+  
+  getName() {
+    if (this.props.dataSource.name) {
+      return this.props.dataSource.name;
+    } else {
+      return this.state.name;
+    }
+  }
+  
+  getCurrentStatus() {
+    let position = this.state.position;
+    let company = this.state.company;
+    
+    const experience = this.props.dataSource.experience;
+    if (experience && experience.length > 0) {
+      company = experience[0].employer.name;
+      if (experience[0].position) {
+        position = experience[0].position.name;
+      }
+      
+      return position + ' at ' + company;
```

Comment:
Nothing shows for description.
![2017-01-04 1 14 18](https://cloud.githubusercontent.com/assets/11007191/21631546/cc386768-d27f-11e6-9729-a7ed30df1194.png)